DirectProgramming/C++/StructuredGrids/iso3dfd_omp_offload/src/iso3dfd.cpp

```diff
@@ -381,7 +381,7 @@ int main(int argc, char *argv[]) {
   }
 
   // Check for available omp offload capable device
-  unsigned int num_devices = omp_get_num_devices();
+  int num_devices = omp_get_num_devices();
```

Comment:
according to OpenMP spec, omp_get_num_devices function returns value with int type.